metadata-ingestion/src/datahub/ingestion/source/tableau/tableau.py

```diff
@@ -1015,13 +1028,13 @@ def get_connection_object_page(
             if retries_remaining <= 0:
                 raise
             return self.get_connection_object_page(
-                query,
-                connection_type,
-                query_filter,
-                count,
-                offset,
+                query=query,
+                connection_type=connection_type,
+                query_filter=query_filter,
+                fetch_size=fetch_size,
+                current_cursor=current_cursor,
                 retry_on_auth_error=False,
-                retries_remaining=retries_remaining - 1,
+                retries_remaining=retries_remaining,
```

Comment:
why aren't we decreasing the retries remaining count?

Reply:
Fixed